setup.py

```diff
@@ -37,5 +37,5 @@
         "write_to": "locust/_version.py",
         "local_scheme": "no-local-version",
     },
-    setup_requires=["setuptools_scm"],
+    setup_requires=["setuptools_scm<=6.0.1"],
```

Comment:
Why this constraint?

Reply:
below the wall of text in the description :)

> while testing the changes i had some problems with the packaging when specifying locust dependency from a git repo.
> i noticed that setuptools-scm [has deprecated using setup.py](https://github.com/pypa/setuptools_scm/#setupcfg-usage-deprecated) after version 6.0.1, so i added a constraint for that package in setup.py. (also, the black configuration specified in pyproject.toml caused problems with pip, causing it to think it was going to use that instead of setup.{py,cfg}, unfortunately the black maintainers https://github.com/psf/black/issues/688).

Reply:
👍